doc/sai-ptf/config_data/config_t0.md

```diff
@@ -0,0 +1,162 @@
+- [IP and MAC naming convention](#ip-and-mac-naming-convention)
+  - [MAC](#mac)
+  - [IP](#ip)
+- [1. L2 Configurations](#1-l2-configurations)
+  - [1.1 FDB Configuration](#11-fdb-configuration)
+  - [1.2 VLAN configuration](#12-vlan-configuration)
+- [2. L3 configuration](#2-l3-configuration)
+  - [2.1 VLAN Interfaces](#21-vlan-interfaces)
+  - [2.2 LAG configuration](#22-lag-configuration)
+    - [2.2.1 LAG Hash Rule](#221-lag-hash-rule)
+  - [2.3 Route Configuration](#23-route-configuration)
+    - [2.3.1 VLAN interfaces route entries](#231-vlan-interfaces-route-entries)
+    - [2.3.2 LAG Route entry](#232-lag-route-entry)
+  - [2.4 Neighbor Configuration](#24-neighbor-configuration)
+    - [2.4.1 VLAN Neighbors](#241-vlan-neighbors)
+    - [2.4.2 LAG Neighbors](#242-lag-neighbors)
+# Overriew
+This document describes the sample configuration data.
+
+**Note: This configuration focused on T0 topology.**
+
+# IP and MAC naming convention
+In this configuration, we mapped the IP and MAC address into different parts of this configuration as below.
+
+## MAC
+For MAC addresses, we can use different sections in the MAC addresses to map different title numbers.
+The pattern is
+```
+L1_NUM:L2_NUM:L3_NUM:ROLE:EXTRA:SEQ
+ROLE: T1=1, Server=99
+```
+
+For example:
+For the MAC address in ``1.1 FDB Configuration``.
+```
+#Server MAC
+01:01:00:99:02:01~01:01:00:99:02:32
+# 01:01：Title 1.1
+# 99: Server
+# 02: EXTRA (Group ID)
+
+#T0 Ports MAC
+01:01:00:00:00:01~01:01:00:00:00:32
+```
+
+For the VLAN Neighbor(Server) MAC in ``2.4.1 VLAN Neighbors``
+```
+02:04:01:00:99:01~02:04:01:00:99:32
+```
+
+## IP
+For IP addresses, we will use different IP subnets for different device roles (only for T0, T1, and server), and map different numbers in the L3 sub-title within the second and third IP sections.
+
+Format: 10.ROLE_NUM+L3_SUB_TITLE_NUM.GROUP_ID*32+L3_SUB2_TITLE_NUM.SEQ
+
+- ROLE_NUM
+T0: 0
+T1: 64
+Server: 128
+
+For example
+```
+#T0 IP in 2.1 VLAN Interface
+#T0 Host interfaces
+10.1.0.1~10.1.0.32
+
+#Server IP in 2.3.1 VLAN interfaces route entries
+#Group0
+10.131.1.1~10.131.1.32
+#Group1
+10.131.33.1~10.131.33.32
+## 131: 128(server) + 3 (L3 first level Sub Title, 3)
+## 33: 32(Group1, 1*32) + 1 (L3 second level Sub Title, 1)
+```
+
+
+
+
+# 1. L2 Configurations
+
+## 1.1 FDB Configuration
+
+The MAC Table for VLAN L2 forwarding as below
+|Name|MAC|PORT|VLAN|HostIf|
+|-|-|-|-|-|
+|mac0|01:01:00:99:00:00|Port0||Ethernet0|
+|mac1-8  |01:01:00:99:01:01 - 01:01:00:99:01:08|Port1-8|10|Ethernet4-Ethernet32|
+|mac9-16 |01:01:00:99:02:09 - 01:01:00:99:02:16|Port9-16|20|Ethernet36-Ethernet64|
+|mac17-mac31 |01:01:00:99:00:17 - 01:01:00:99:00:31|Port17-31||Ethernet68-Ethernet124|
+
+## 1.2 VLAN configuration
+
+|HostIf|VLAN ID|Ports|Tag mode|
+|-|-|-|-|
+|Ethernet4-32|10|Port1-8|Untag|
+|Ethernet36-72|20|Port9-16|Untag|
+
+
+# 2. L3 configuration
+
+Host interface IP
+|Port|VLAN Interface IP| 
+|-|-|
+|port0|10.0.0.100|
+|port1-31|10.0.0.1-10.0.0.31|
```

Comment:
no ip address for layer 2 ports. @richardyu-ms

Reply:
removed